src/BenchmarkDotNet/Toolchains/NativeAot/Generator.cs

```diff
@@ -190,18 +188,10 @@ protected override void GenerateProject(BuildPartition buildPartition, Artifacts
 </Project>";
 
         private string GetTrimmingSettings()
-        {
-            StringBuilder sb = new StringBuilder();
-
-            sb.AppendLine(rootAllApplicationAssemblies ? "<PublishTrimmed>false</PublishTrimmed>" : "<TrimMode>link</TrimMode>");
-
-            if (!string.IsNullOrEmpty(trimmerDefaultAction))
-            {
-                sb.Append($"<TrimmerDefaultAction>{trimmerDefaultAction}</TrimmerDefaultAction>");
-            }
-
-            return sb.ToString();
-        }
+            => rootAllApplicationAssemblies
+                ? "" // use the defaults
+                // TrimMode is set in explicit way as for older versions it might have different default value
```

Comment:
`TrimMode=link` by default in .NET 6 and NativeAOT doesn't support .NET 5. It's probably unnecessary.

Reply:
We are still supporting .NET 5 with runtimelab ILCompiler (`--runtimes nativeaot5.0`) and I've noticed that it has great impact on the build time for this particular config (which is used by our CI as we are still using .NET 5 SDK in BDN)

Reply:
The ILCompiler package brings it's own compiler, CoreLib and the entire BCL - I think if you use the runtimelab ILCompiler with `--runtimes nativeaot5.0` or `--runtimes nativeaot6.0`, you won't see any difference in their performance. The only difference is that the reference assemblies that the C# compiles against won't let you use 6.0 APIs with runtimes=5.0. Either way you get RyuJIT, runtime, CoreLib and framework that is half way between 6.0 and 7.0.

The only difference might be if there's third party NuGets - it will pick up 5.0 versions if they multitarget.

Reply:
After this one https://github.com/dotnet/BenchmarkDotNet/pull/1978 does `--runtimes nativeaot5.0` needed?

Reply:
I've removed `nativeaot5.0`. Moreover, I think that once we ship .NET 7 with official NativeAOT support we should also remove `nativeaot6.0`.

BTW I've tested the following command:

```cmd
dotnet run -c Release -f net6.0 --filter *Basic* --job dry --runtimes nativeaot6.0
```

With `<TrimMode>link</TrimMode>` it takes 9s to publish the generated app. Without it it takes 110s. So I am going to keep it.